packages/foreman/foreman/foreman.spec

```diff
@@ -763,6 +764,22 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
 
+# SELinux libexec wrappers
+%if 0%{?scl:1}
+cat > %{buildroot}%{_libexecdir}/%{name}/sidekiq-selinux <<EOF
+#!/bin/bash
+# Shell wrapper with SELinux transition into foreman_rails_t domain.
+source scl_source enable %{scl}
+exec sidekiq "\$@"
+EOF
+%else
+cat > %{buildroot}%{_libexecdir}/%{name}/sidekiq-selinux <<EOF
+#!/bin/bash
+# Shell wrapper with SELinux transition into foreman_rails_t domain.
+exec sidekiq "\$@"
+EOF
+%endif
```

Comment:
Wouldn't this also work:

```suggestion
# SELinux libexec wrappers
cat > %{buildroot}%{_libexecdir}/%{name}/sidekiq-selinux <<EOF
#!/bin/bash
# Shell wrapper with SELinux transition into foreman_rails_t domain.
%if 0%{?scl:1}
source scl_source enable %{scl}
%endif
exec sidekiq "\$@"
EOF
```

Reply:
Yeah constant refactoring of a specific code makes you blind.